style.css

```diff
@@ -39,3 +39,40 @@
 .next.item.icon {
   font-size: 2em;
 }
+
+.calendar {
+  table-layout: fixed!important;
+}
+
+.calendar td {
+  display: table-cell!important;
+}
+
+.disabled {
+  background-color: rgba(177, 177, 177, 0.651)!important;
+}
+
+.disabled:hover {
+  cursor:not-allowed;
+}
+
+.month-next {
+  float: right;
+}
+
+.month-previus {
+  float: left;
+}
+
+caption {
```

Comment:
ehhez a stílhez is a kérdés, hogy nem votl valami ilyesmi klassz vagy klasszok bootstrapben? (nagyobb, közép)